Library/Homebrew/test/test_integration_cmds.rb

```diff
@@ -257,28 +290,13 @@ def test_tap
   end
 
   def test_missing
-    repo = CoreTap.new
-    foo_file = repo.formula_dir/"foo.rb"
-    foo_file.write <<-EOS.undent
-      class Foo < Formula
-        url "https://example.com/foo-1.0"
-      end
-    EOS
-
-    bar_file = repo.formula_dir/"bar.rb"
-    bar_file.write <<-EOS.undent
-      class Bar < Formula
-        url "https://example.com/bar-1.0"
-        depends_on "foo"
-      end
-    EOS
+    setup_test_formula "foo"
+    setup_test_formula "bar"
```

Comment:
Would it be preferable to use `setup_test_formula "foo" && setup_test_formula "bar"` instead of
```
setup_test_formula "foo"
setup_test_formula "bar"
```
?